pkg/services/grafana-apiserver/README.md

```diff
@@ -32,10 +32,12 @@ etcd_servers = 127.0.0.1:2379
 
 ### `kubectl` access
 
-From the root of the repository:
-
+1. `kubectl` requires HTTPS, so you will need to [configure and enable HTTPS](https://grafana.com/docs/grafana/latest/setup-grafana/set-up-https/) in `custom.ini`.
```

Comment:
@mildwonkey @charandas `kubectl` access becomes more complicated with this change because HTTPS is required. does this seem acceptable?

Reply:
both acceptable and preferable, imo!